django/utils/log.py

```diff
@@ -27,7 +27,8 @@
     'formatters': {
         'django.server': {
             '()': 'django.utils.log.ServerFormatter',
-            'format': '[%(server_time)s] %(message)s',
+            'format': '[{server_time} {message}',
```

Comment:
Tests are failing because you dropped the `]`.

Reply:
Updated.